Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/exceptions.py

```diff
@@ -0,0 +1,18 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Exceptions that can occur while interacting with a Launcher
+"""
+
+
+class LyTestToolsFrameworkException(Exception):
```

Comment:
This file ideally should be declared at the sub-module which consumes it. It is not specific to pytest_plugin, so it should be at least one directory upward.

I could see arguments for existing at `ly_test_tools/_internal/` (suggesting it should not be used by customer code) or in `ly_test_tools/`